Automattic/wp-calypso
# Jetpack AI Assistant: Add Jetpack AI green product icon on the My Plan page

Related to #

## Proposed Changes

* Add support for the green product icon of Jetpack AI.

## Testing Instructions



* Spin this change with the Live Calypso link.
* Navigate to the My Plan page for a site with purchasing Jetpack AI: `/plans/my-plan/{site-slug}`.
* Ensure the icon of Jetpack AI works as expected.

<img width="996" alt="截圖 2023-07-20 下午11 39 01" src="https://github.com/Automattic/wp-calypso/assets/6869813/078f57c3-e8c3-4aa3-8aa1-986e09e5711c">

## Pre-merge Checklist
